src/SpotifyApi.NetCore/FollowApi.cs

```diff
@@ -11,102 +12,97 @@ namespace SpotifyApi.NetCore
+        /// Check if Current User Follows Artists or Users
         /// </summary>
-        /// <param name="ids">	Required. A comma-separated list of the artist or the user Spotify IDs to check. For example: ids=74ASZWbe4lXaubB36ztrGX,08td7MxkoHQkXnWAYD8d6Q. A maximum of 50 IDs can be sent in one request.</param>
+        /// <param name="artistIds">Required. A comma-separated list of the artist Spotify IDs. A maximum of 50 artist IDs can be sent in one request. A minimum of 1 artist id is required.</param>
         /// <returns>bool[] an array of true or false values, in the same order in which the ids were specified.</returns>
         /// <remarks>
         /// https://developer.spotify.com/documentation/web-api/reference/follow/check-current-user-follows/
         /// </remarks>
         public async Task<T> CheckCurrentUserFollowsArtists<T>(
-            string[] ids,
+            string[] artistIds,
             string accessToken = null
             )
         {
-            if (ids?.Length < 1 || ids?.Length > 50) throw new ArgumentNullException("ids");
+            if (artistIds?.Length < 1 || artistIds?.Length > 50) throw new
+                    ArgumentException("A minimum of 1 and a maximum of 50 artist ids can be sent.");
 
-            var builder = new UriBuilder($"{BaseUrl}/me/following/contains");
+            UriBuilder builder = new UriBuilder($"{BaseUrl}/me/following/contains");
             builder.AppendToQuery("type", "artist");
-            builder.AppendToQueryAsCsv("ids", ids);
+            builder.AppendToQueryAsCsv("ids", artistIds);
             return await GetModel<T>(builder.Uri, accessToken);
         }
 
         /// <summary>
-        /// Check if Current User Follows Users
+        /// Check if Current User Follows Artists or Users
         /// </summary>
-        /// <param name="ids">	Required. A comma-separated list of the artist or the user Spotify IDs to check. For example: ids=74ASZWbe4lXaubB36ztrGX,08td7MxkoHQkXnWAYD8d6Q. A maximum of 50 IDs can be sent in one request.</param>
+        /// <param name="userIds">Required. A comma-separated list of the user Spotify IDs. A maximum of 50 user IDs can be sent in one request. A minimum of 1 user id is required.</param>
         /// <returns>bool[] an array of true or false values, in the same order in which the ids were specified.</returns>
         /// <remarks>
         /// https://developer.spotify.com/documentation/web-api/reference/follow/check-current-user-follows/
         /// </remarks>
         public async Task<bool[]> CheckCurrentUserFollowsUsers(
-            string[] ids,
+            string[] userIds,
             string accessToken = null
-            ) => await CheckCurrentUserFollowsUsers<bool[]>(ids, accessToken);
+            ) => await CheckCurrentUserFollowsUsers<bool[]>(userIds, accessToken);
 
         /// <summary>
-        /// Check if Current User Follows Users
+        /// Check if Current User Follows Artists or Users
```

Comment:
"Check to see if the current user is following one or more other Spotify users."